# This patch has been taken from the Debian bug tracker found at
# http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=288175
# resolving compile problems with GCC 4, see bug #141292
--- /tmp/charset.c	2006-07-25 09:30:28.000000000 +0200
+++ charset.c	2006-07-25 09:31:18.000000000 +0200
@@ -20,7 +20,7 @@
 
 QECharset *first_charset = NULL;
 
-extern QECharset charset_7bit;
+static QECharset charset_7bit;
 
 /* specific tables */
 static unsigned short table_idem[256];
